src/test/java/TestSuite.java

```diff
@@ -0,0 +1,14 @@
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.platform.suite.api.SelectClasses;
+import org.junit.runner.RunWith;
+
+@RunWith(JUnitPlatform.class)
```

Comment:
Am I correct that @SelectClasses is an annotation from Unit 4? As far as we are using Unit 5, please, consider nested test classes mechanism https://junit.org/junit5/docs/current/user-guide/#writing-tests-nested instead of @SelectClasses.